src/test/java/edu/cornell/kfs/module/cam/document/service/impl/CuAssetServiceImplTest.java

```diff
@@ -1,42 +1,124 @@
 package edu.cornell.kfs.module.cam.document.service.impl;
 
 import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
 
+import static org.junit.Assert.assertFalse;
```

Comment:
If this static import isn't being used by the test, then I would suggest removing it.